src/Command/SelfUpdateCommand.php

```diff
@@ -86,7 +101,7 @@ protected function configure(): void
             null,
             InputOption::VALUE_REQUIRED,
             'The channel of the release. Right now only unstable is available',
```

Comment:
This comment does not make sense anymore - we default to `stable` and say that only `unstable` is available.